src/entity/User.java

```diff
@@ -1,4 +1,19 @@
 package entity;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+
 public interface User {
```

Comment:
Follows interface segregation design principle by not defining irrelevant methods.